### PR TITLE
Add data tables to neonatal sepsis model

### PR DESCRIPTION
**Pull request by kiranklc**

### CODE REVIEW

File: docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst [hunk 1]
```diff
@@ -18,3 +18,137 @@ pneumonia as a separate entity.
...
+     - disabilty weights
+     - 0
+     -
+   * - I
+     - prevalence
+     - prevalence_c383
+     - 
+   * - I
+     - birth prevalence
+     - 1-birth_prevalence_c383
```
Comment (aflaxman) [inline docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst]: Typo?  I was expecting `birth_prevalence_c383` here, not one minus it

File: docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst [hunk 2]
```diff
@@ -18,3 +18,137 @@ pneumonia as a separate entity.
...
+   * - Transition
+     - Source 
+     - Sink 
+     - Value
+     - Notes
+   * - i
+     - S
+     - I
+     - :math:`\frac{\text{incidence_rate_m1594}}{\text{1 - prevalence_c383}}`
+     - Incidence rate is estimated in dismod but does not exists in como.
```
Comment (aflaxman) [inline docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst]: If incidence rate comes from dismod, I think we don't need to divide through by prevalence.

File: docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst (hunk 1 above)
Comment (kiranklc) [reply docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst]: Sorry, its a typo.. I copied from the previous row and forgot to change it. I will modify it.

File: docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst (hunk 2 above)
Comment (kiranklc) [reply docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst]: I will change it.

**Pull request merged**